text/0118-scoped-slots-light-dom.md

```diff
@@ -9,6 +9,33 @@ pr:
 
 # Scoped Slots in Light DOM
 
+Table of Content:
+* [Summary](#summary)
+* [Motivation](#motivation)
+* [Prior Art](#prior-art)
+    * [Vue.js](#vue.js)

```

Comment:
```suggestion
    * [Vue.js](#vuejs)
```

Reply:
I verified that it is indeed #vue.js

![Screen Shot 2022-09-28 at 4 38 16 PM](https://user-images.githubusercontent.com/931465/192906992-ce600213-ae54-4c7f-802f-e758214f9115.png)